CMakeLists.txt

```diff
@@ -1,17 +1,13 @@
-cmake_minimum_required(VERSION 3.12.0 FATAL_ERROR)
+cmake_minimum_required(VERSION 3.11.0 FATAL_ERROR)
 
-option(IRODS_ENABLE_430_COMPATIBILITY "Enables compatibility with iRODS 4.3.0." YES) # Default to YES for community.
-
-find_package(IRODS REQUIRED CONFIG)
+find_package(IRODS 4.2.11 REQUIRED CONFIG)
```

Comment:
My understanding is that this *requires* 4.2.11 packages, meaning, this will not work with 4.2.12 packages. Is that what we want? I am not sure about my understanding here...

Reply:
This is what I'm investigating at the moment.

I want to set 4.2.11 as the minimum version required so no one has to touch the cmake.

I'm pretty sure this didn't work earlier with 4.2.12, but I'm going to give it one more try before I switch to another way of handling this.

Reply:
Oh, I see. I didn't understand your comment from before. Carry on!

Reply:
It's *supposed* to be a minimum unless you specify `EXACT`.

Reply:
Right. I think I got confused because, generally speaking, our packages are not always (or even usually? or perhaps just historically?) compatible across different versions.